lib/stripe.js

```diff
@@ -133,6 +135,33 @@ Stripe.prototype = {
     );
   },
 
+  setAppInfo: function(info) {
+    if (info && typeof info !== 'object') {
+      throw new Error('AppInfo must be an object.');
+    }
+
+    info = info || {};
+
+    var appInfo = APP_INFO_PROPERTIES.reduce(function(accum, prop) {
+      if (typeof info[prop] == 'string') {
+        accum = accum || {};
+
+        accum[prop] = info[prop];
+      }
+
+      return accum;
+    }, undefined);
+
+    if (appInfo && Object.keys(appInfo).length > 0 && !appInfo['name']) {
```

Comment:
Nice! What do you think about tweaking this a bit so that it's simply "must include name"? It seems that calling `setAppInfo` with an empty object is fairly counterproductive and we may as well error on it.